### DISCUSSION TITLE
How can I add a helm chart repo that I don't own to artifacthub.io?

### DISCUSSION THREAD

**Discussion by ecerulm** [artifacthub/hub#3962] [Q&A]
The "official" aws-efs-csi-driver helm chart is missing from artifacthub.io

* https://kubernetes-sigs.github.io/aws-efs-csi-driver/
* `helm repo add aws-efs-csi-driver https://kubernetes-sigs.github.io/aws-efs-csi-driver/`

How do I request it to be added to artifacthub.io, I'm not the owner or affiliated with the project.

**Comment by tegioz** [comment]
Hi @ecerulm 👋 

Any user or organization can publish any repository to artifacthub.io. In cases like this, the best approach is to contact the software owners (maybe by opening an issue in their source repository) to suggest them to publish the content on artifacthub.io. If this doesn't work, you can always publish it yourself. The legit owner will be able to [claim its ownership](https://artifacthub.io/docs/topics/repositories/#ownership-claim) later on if they wish so.

It looks like in this case [someone has already published this repository](https://artifacthub.io/packages/helm/aws-ebs-csi-driver/aws-ebs-csi-driver). As mentioned above, the legit owner can claim its ownership in an automated way at any time (and obtain the [verified publisher](https://artifacthub.io/docs/topics/repositories/#verified-publisher) status -and potentially request the [official status](https://artifacthub.io/docs/topics/repositories/#official-status) when applicable-).
